enterprise/app/trends/drilldown_page.tsx

```diff
@@ -506,6 +506,7 @@ export default class DrilldownPageComponent extends React.Component<Props, State
   handleClearZoom() {
     router.setQuery({
       ...Object.fromEntries(this.props.search.entries()),
+      [DD_SELECTED_AREA_URL_PARAM]: "",
```

Comment:
This field doesn't seem to be used anywhere. Maybe this PR is missing the changes for "Clear the active selection when zoom is cleared."? Don't see changes for that anywhere

Reply:
This is already read at L466: https://github.com/buildbuddy-io/buildbuddy/blob/a116cb72b76a3c3b8e22f92c7209841bd8bbd345/enterprise/app/trends/drilldown_page.tsx#L466

This change is just fixing a case where this field wasn't getting reset

Reply:
Ah my bad - didn't have master pulled in my local branch